lib/OpenQA/WebAPI/Controller/API/V1/Job.pm

```diff
@@ -602,10 +603,10 @@ sub create_artefact {
             });
     }
     if ($job->create_artefact($validation->param('file'), $self->param('ulog'))) {
-        $self->render(text => "OK");
+        $self->render(text => 'OK');
     }
     else {
-        $self->render(text => "FAILED");
+        $self->render(json => {error => 'Unable to create artefact'}, status => 500);
```

Comment:
Should it worry me that existing tests aren't failing? I would think we want to catch at least one occurence of it

Reply:
Since I wasn't sure where to extend tests I've just been creating the PR in the hope the CI checks will tell me. The API tests are actually failing so you shouldn't be worried :-)